build.gradle

```diff
@@ -53,3 +54,21 @@ check.dependsOn jacocoTestReport
 jacocoTestReport {
     reports.xml.enabled true
 }
+
+publishing {
+    repositories {
+        maven {
+            name = "GitHubPackages"
+            url = uri("https://maven.pkg.github.com/Aalto-LeTech/intellij-plugin")
+            credentials {
+                username = project.findProperty("gpr.user") ?: System.getenv("GRP_USER")
+                password = project.findProperty("gpr.key") ?: System.getenv("GRP_KEY")
```

Comment:
In which situations we use each of these (project property vs. env variable)?

Reply:
this is just an artifact from the example, I'll remove!